test/e2e/cypress/e2e/host_details.cy.js

```diff
@@ -355,4 +355,41 @@ context('Host Details', () => {
       cy.get('span').find('svg').should('exist');
     });
   });
+
+  describe('Deregistration', () => {
+    describe('"Clean up" button should be visible only for an unhealthy host', () => {
+      it('should not display the "Clean up" button for healthy host', () => {
+        cy.task('startAgentHeartbeat', [selectedHost.id]);
+        cy.contains('button', 'Clean up').should('not.exist');
+      });
+
+      it('should show the "Clean up" button once heartbeat is lost and debounce period has elapsed', () => {
+        cy.task('stopAgentsHeartbeat');
+        cy.contains('button', 'Clean up', { timeout: 15000 }).should('exist');
+      });
+    });
+
+    describe('"Clean up" button should deregister a host', () => {
+      before(() => {
+        cy.task('stopAgentsHeartbeat');
+      });
+
+      it('should allow to deregister a host after clean-up confirmation', () => {
+        cy.contains('button', 'Clean up', { timeout: 15000 }).click();
```

Comment:
The value of `timeout` depends on Elixir environment variable [`config :trento, deregistration_debounce`](https://github.com/trento-project/web/blob/main/config/test.exs#L49). These timeouts are also present in the [Hosts List tests](https://github.com/trento-project/web/blob/main/test/e2e/cypress/e2e/hosts_overview.cy.js#L205) for the same reason. Could we abstract this to an environment variable somehow?